subscription_oca/views/sale_subscription_views.xml

```diff
@@ -370,12 +371,66 @@
         <field name="model">sale.subscription</field>
         <field name="arch" type="xml">
             <search>
+                <field
+                    name="name"
+                    filter_domain="['|', ('name', 'ilike', self), ('code', 'ilike', self)]"
+                />
+                <field
+                    name="commercial_partner_id"
+                    filter_domain="[
+                        '|', ('partner_id', 'ilike', self), ('commercial_partner_id', 'ilike', self)]"
+                />
+                <separator />
+                <filter
+                    string="My Subscriptions"
+                    domain="[('user_id', '=', uid)]"
+                    name="my_subscription_filter"
+                />
+                <separator />
+                <filter
+                    string="Draft / Waiting"
```

Comment:
This filter also shows the closed subscriptions.

Maybe we should add the related stage_type field to the subscription and create better filters?